README.md

```diff
@@ -155,6 +155,7 @@ These variables are used by the service startup scripts in the Docker images, bu
 * `SERVER_ID_IS_HOSTNAME`: If set to "true", then instead of generating random UUIDs as the server ID when adding a server Scalelite will use the hostname of the server as the id. Server hostnames will be checked for uniqueness. Defaults to "false".
 * `CREATE_EXCLUDE_PARAMS`: List of BBB server attributes that should not be modified by create API call. Should be in the format 'CREATE_EXCLUDE_PARAMS="param1,param2,param3"'.
 * `JOIN_EXCLUDE_PARAMS`: List of BBB server attributes that should not be modified by join API call. Should be in the format 'JOIN_EXCLUDE_PARAMS="param1,param2,param3"'.
+* `RECORDING_READY_NOTIFIER_TIME`: Schedule recording ready notification to run after x minutes. Defaults to 2 minutes'.
```

Comment:
Why is this delay needed?, what happen if I set it up to 0?

Reply:
I agree - no need for this setting to exist, just enqueue the notification to run immediately after the recording is published.